Formula/lib/libmps.rb

```diff
@@ -0,0 +1,41 @@
+class Libmps < Formula
+  desc "Memory Pool System"
+  homepage "https://www.ravenbrook.com/project/mps/"
+  url "https://github.com/Ravenbrook/mps/archive/refs/tags/release-1.118.0.tar.gz"
+  sha256 "58c1c8cd82ff8cd77cc7bee612b94cf60cf6a6edd8bd52121910b1a23344e9a9"
+  license "BSD-2-Clause"
+
+  depends_on xcode: :build
+  depends_on :macos
+
+  def install
+    # Build the project using xcodebuild
+    xcodebuild  "-scheme", "mps",
```

Comment:
Why don't we use `configure` and `make` so that it doesn't have to be just for macOS?

Reply:
I wrote the following code according to the guide. [mps build guide](https://github.com/Ravenbrook/mps/blob/master/manual/build.txt)

```ruby
  depends_on "autoconf" => :build
  depends_on "automake" => :build
  depends_on "libtool" => :build

  def install
    system "autoreconf"
    system "./configure", "--prefix=#{prefix}"
    system "make"
    system "make", "install"
  end
```

The original repository of mps will build universal binaries, The audit tool warned me:

```
➜ HOMEBREW_NO_INSTALL_FROM_API=1 brew audit --new --strict mps
Warning: Treating mps as a formula. For the cask, use homebrew/cask/mps or specify the `--cask` flag.
mps
  * Unexpected universal binaries were found.
    The offending files are:
      /opt/homebrew/Cellar/mps/1.118.0/bin/mpseventcnv
      /opt/homebrew/Cellar/mps/1.118.0/bin/mpseventpy
      /opt/homebrew/Cellar/mps/1.118.0/bin/mpseventsql
      /opt/homebrew/Cellar/mps/1.118.0/bin/mpseventtxt
Error: 1 problem in 1 formula detected.
```

perhaps I need to learn more about `configure` and `make` related knowledge 😄.

Reply:
Ah, that's because `./configure && make` just runs `xcodebuild` on macOS. We can proceed as you've done here with calling `xcodebuild` directly on macOS and `configure` on Linux.

Reply:
`./configure && make` will build binary files, which I think:

- are not necessary as a library.
- but deleting these files could be risky.
- MPS build guide doesn't recommend using configure & make, which makes me more confused.

Ah, we can handle this separately for macOS and Linux.